dask_cloudprovider/aws/ecs.py

```diff
@@ -755,6 +768,11 @@ def __init__(
         self._worker_mem = worker_mem
         self._worker_gpu = worker_gpu
         self._worker_extra_args = worker_extra_args
+        self._worker_task_def_name = (
+            worker_task_def_name
+            if worker_task_def_name
+            else "dask-worker-{}".format(str(uuid.uuid4())[:5])
+        )
```

Comment:
I wonder if we can switch this up so we for the formatting after it chooses either the user config or the default. That was a user could pass a task def name like `"my-awesome-task-def-{uuid}"` and it will still be populated with a UUID.

```suggestion
        self._worker_task_def_name = (
            worker_task_def_name
            if worker_task_def_name
            else "dask-worker-{uuid}"
        ).format(uuid=str(uuid.uuid4())[:5])
        
```

Reply:
that makes sense